assets/js/base/context/hooks/collections/use-collection-data.ts

```diff
@@ -38,6 +42,14 @@ export const useCollectionData = ( {
 	queryPrices,
 	queryStock,
 	queryState,
+}: {
```

Comment:
I would suggest pulling this out into a type to save space here. As a general rule, if the type declarations are over 3-4 lines long, I normally like to pull them out as it makes scanning the code easier. That's just preference though, not a blocker!

Reply:
Make sense to me too! I applied your suggestion. Thanks so much!